rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/project/ProjectController.java

```diff
@@ -579,9 +582,14 @@ public ResponseEntity<CollectionModel<EntityModel<Release>>> getReleasesOfLinked
     @RequestMapping(value = PROJECTS_URL + "/{id}", method = RequestMethod.DELETE)
     public ResponseEntity deleteProject(
             @Parameter(description = "Project ID")
-            @PathVariable("id") String id
+            @PathVariable("id") String id,
+            @Parameter(description = "Comment message")
+            @RequestParam(value = "comment", required = false) String comment
```

Comment:
The `comment` parameter is not used here. Missed `sw360User.setCommentMadeDuringModerationRequest(comment);` ?